pkg/mapper/remove.go

```diff
@@ -24,8 +24,6 @@ import (
 
 // Remove removes by match of provided arguments
 func (b *AuthMapper) Remove(args *RemoveArguments) error {
-	args.Validate()
```

Comment:
I think just having a client-side validation is not enough. Do you think we can bring this back here?

Reply:
Good point, I will move those back

Reply:
fixed